### PR TITLE
Add default sbb consent management

### PR DESCRIPTION
**Pull request by oterral**
# How to

Disply the official SBB consent window if enableTracking is true.

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [x] The images added are optimized.
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [x] The title means something for a human being.
- [x] The title contains [WIP] if it's necessary.
- [x] Labels applied. if it's a release? a hotfix?
- [x] Tests added.

### CODE REVIEW

File: cypress/integration/Search.spec.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ describe('Search', () => {
     cy.visit('');
   });
   it(`should display plural texts`, () => {
+    cy.get('#onetrust-accept-btn-handler').click();
```
Comment (friedjoff) [inline cypress/integration/Search.spec.js:8]: Maybe move into `beforeEach`?

File: src/WebComponent.js [hunk 2]
```diff
@@ -133,22 +133,12 @@ const propTypes = {
   /**
    * Enable analytics tracking.
    */
-  enableTracking: PropTypes.bool,
+  enableTracking: PropTypes.string,
```
Comment (friedjoff) [inline src/WebComponent.js:136]: String because of web component?

File: src/components/Head/Head.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+import Helmet from 'react-helmet';
+
+/**
+ * This component add tag in the htm head tag for sbb consent management.
```
Comment (friedjoff) [inline src/components/Head/Head.js:7]:
```suggestion
 * This component adds a script tag in the HTML head for SBB consent management.
```

File: src/components/MatomoTracker/MatomoTracker.js [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+  useEffect(() => {
+    if (!activeTopicKey || !consentGiven || !trackEvent || !localStorage) {
+      return;
+    }
+    const startDate = localStorage?.getItem(LS_MATOMO_USER_SESSION_TIMER) || 0;
+    const topicVisited = localStorage?.getItem(LS_MATOMO_TOPIC_VISITED) || '';
+
+    // A matomo user session is 30 min by default.
+    if (
+      Date.now() - startDate > 30 * 60 * 1000 ||
```
Comment (friedjoff) [inline src/components/MatomoTracker/MatomoTracker.js:45]: Maybe extract `30 * 60 * 1000` into a `const` variable?

**Review by friedjoff — COMMENTED** [review]
Code looks solid! Just small suggestions...

File: src/WebComponent.js (hunk 2 above)
Comment (oterral) [reply src/WebComponent.js:136]: yes when you specify an attribute in the webcomponent it has to be string. A property can be somthing else but not an attribute.

File: cypress/integration/Search.spec.js (hunk 1 above)
Comment (oterral) [reply cypress/integration/Search.spec.js]: Done

**Pull request merged**